beacon-chain/blockchain/service.go

```diff
@@ -8,9 +8,11 @@ import (
 	"fmt"
```

Comment:
Please review this file - this changes the blockchain service to use a single state